assets/js/script.js

```diff
@@ -0,0 +1,17 @@
+let currentTimeDoc = document.getElementById('currentTime');
```

Comment:
Use `const` by default.
You are not going to change it.

Reply:
I created new PR with this changings, thank you! Should i merge this new PR and create new branch for new task?